BhuTuka Expanded One: Version 1.000; ttfautohint (v1.8.3) added

yanone opened the pull request:
e530d4e: [gftools-packager] BhuTuka Expanded One: Version 1.000; ttfautohint (v1.8.3) added

* BhuTuka Expanded One Version 1.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/erinmclaughlin/BhuTuka-Extended-One at commit https://github.com/erinmclaughlin/BhuTuka-Extended-One/commit/ac2ad17bcd23da70b2c63a4ed794cbb7a7ebaac6.

 783d8e8: [gftools-packager] ofl/bhutukaexpandedone remove METADATA "source".  google/fonts#2587

yanone commented:
Please ignore the FAILs about the name as that's intentional. I had asked Felipe to add the name to the exceptions in fontbakery weeks ago.

RosaWagner commented:
Can you update gftools (and packager) and re-package? There should be lang tags in the metadata.pb

yanone commented:
Updated

## BhuTuka Expanded One: Version 1.000; ttfautohint (v1.8.3) added

---
 87b8cd6: [gftools-packager] BhuTuka Expanded One: Version 1.000; ttfautohint (v1.8.3) added

* BhuTuka Expanded One Version 1.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/erinmclaughlin/BhuTuka-Extended-One at commit https://github.com/erinmclaughlin/BhuTuka-Extended-One/commit/ac2ad17bcd23da70b2c63a4ed794cbb7a7ebaac6.

 e0cea5b: [gftools-packager] ofl/bhutukaexpandedone remove METADATA "source".  google/fonts#2587
